oneview_redfish_toolkit/blueprints/manager.py

```diff
@@ -41,41 +43,36 @@ def get_managers(uuid):
         /redfish/v1/Managers/id is requested.
 
         Returns:
-            JSON: JSON with Managers info for Enclosure or ServerHardware.
+            JSON: JSON with Managers info.
     """
     try:
-        appliance_information = \
-            g.oneview_client.appliance_node_information.get_version()
-        # TODO(victorhugorodrigues): Handle multiple OneViews
-        oneview_version = appliance_information[0]['softwareVersion']
+        state_url = "/controller-state.json"
+        ov_health_status_url = "/rest/appliance/health-status"
 
-        resource_index = g.oneview_client.index_resources.get_all(
-            filter='uuid=' + uuid
+        ov_ip = get_oneview_ip_by_manager_uuid(uuid)
+        ov_client = client_session.get_oneview_client(ov_ip)
+
+        ov_appliance_info = multiple_oneview.execute_query_ov_client(
+            ov_client, "appliance_node_information", "get_version"
+        )
+        ov_appliance_state = multiple_oneview.execute_query_ov_client(
+            ov_client, "connection", "get", state_url
+        )
+        ov_appliance_health_status = multiple_oneview.execute_query_ov_client(
+            ov_client, "connection", "get", ov_health_status_url
```

Comment:
Nice direct calls reusing the methods on multiple_oneview, this way we keep centralized all OneView request.
Maybe in the future we could keep in request scope which OneView client that request is using for our GET request in a single resource, not for collection. This change could help performance on our GET request for just one resource.